user/server-grpc-example.md

```diff
@@ -144,6 +145,11 @@ func main() {
 	if err != nil {
 		os.Exit(1)
 	}
+
+	if resp.Status != protocol.Ok {
```

Comment:
Updated it to include an error message, also making it explicit that `resp.Errors` might contain multiple errors.